app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt

```diff
@@ -34,9 +34,7 @@ fun DangerousActionDialogue(
     onDismiss: () -> Unit,
     confirmText: String = stringResource(R.string.component_dialogue_dangerous_action_button_confirm),
     dismissText: String = stringResource(R.string.component_dialogue_dangerous_action_button_dismiss),
-    colorContainer: Color = MaterialTheme.colorScheme.surface,
-    colorButtonDismiss: Color = DefaultButton,
-    colorButtonConfirm: Color = Color.Red,
```

Comment:
Incorrect change, and somewhat improperly applied. This needs to be rolled back. Remember that if a function is using input parameters, much like the dangerous action dialogue, there's often a reason for it. Now you've practically hard-coded the colors, which means that calls to it can no longer override default colors if necessary. In this case, you should have added the colors that would need to have been changed as the new default value of the parameter list and not _remove_ the parameters.